src/triggers/CBL_Attitud_UpdateDataCbl2.trigger

```diff
@@ -0,0 +1,49 @@
+trigger CBL_Attitud_UpdateDataCbl2 on CBL_Attitude__c (after insert) {
+    Set<Id> att_Ids = new Set<Id>();
+    Set<Id> cbl_Ids = new Set<Id>();
+    
+    //get a list with Baseline id related to attitude rows inserted
+    for (CBL_Attitude__c newRow : Trigger.new) {
+        att_Ids.add(newRow.Id);
+        cbl_Ids.add(newRow.Baseline__c);
+    }
+    
+    //get a list with attitude rows completed (field Baseline__r.Farmer__c)
+    for(List<CBL_Attitude__c> atts : [SELECT Id, Baseline__c, Baseline__r.Farmer__c, chooseToBeFarmer__c, destinedToBeFarmer__c, discussMethodsWOthers__c, effortInAgriculturePays__c, farmersLifeWillBeBetter__c, futureCoffeePlans__c, happyToBeFarmer__c, limitationsCauses__c, neighborsProductivityLvl__c, payForAgriAdvise__c, productivityImprovmntLimitations__c, productivityTrend5Years__c, takeAdvise__c, wishChildrenBecomeFarmers__c, worthInvestingMoneyTimeAgr__c, Submission__c 
```

Comment:
Ah Ok. I get it. It seems to have been because the pullrequest had conflicts and merging was disabled